src/main/java/com/devcomanda/easymeetup/service/MeetupService.java

```diff
@@ -0,0 +1,7 @@
+package com.devcomanda.easymeetup.service;
+
+import com.devcomanda.easymeetup.entity.Meetup;
+
+public interface MeetupService {
+    Meetup saveOrUpdateMeetup(Meetup meetUp);
```

Comment:
The current task is about creating a new meetup,
I suggest you don't think about updating now.
A simple name like create will be better for this operation. One method -  one thing